Library/Homebrew/build.rb

```diff
@@ -105,7 +105,8 @@ def install
 
     formula.extend(Debrew::Formula) if ARGV.debug?
 
-    formula.brew do
+    formula.brew do |f, stg|
```

Comment:
About `stage`: I see. Thanks for clarifying! `stage_context` is fine with me. I'm happy with almost everything that is less cryptic than `stg`. :smile_cat: